src/parser.rs

```diff
@@ -258,14 +258,15 @@ impl<'a> Parser<'a> {
 
             // blank
             if line.is_empty() {
-                if !item.snippet.is_empty() {
+                if !item.snippet.is_empty() && inside_snippet {
                     item.snippet.push_str(deser::LINE_SEPARATOR);
                 }
             }
             // tag
             else if line.starts_with('%') {
                 should_break = self.write_cmd(&item).is_err();
                 item.snippet = String::from("");
```

Comment:
Can someone enlighten me on why we reset the item rather than creating a new item object ?